PowerEditor/src/WinControls/ProjectPanel/ProjectPanel.cpp

```diff
@@ -784,6 +1289,64 @@ HTREEITEM ProjectPanel::addFolder(HTREEITEM hTreeItem, const TCHAR *folderName)
 	return addedItem;
 }
 
+HTREEITEM ProjectPanel::addDirectory(HTREEITEM hTreeItem, const TCHAR *folderName, bool root, const TCHAR *monitorPath, const std::vector<generic_string>* filters)

```

Comment:
I'm sorry for misrepresenting your position. I'm actually quite glad that I'm wrong here. This is very different than the "no refactoring/cleanup PRs will be accepted!"  that I've previously encountered.